dist/gcc-compatible/EverCrypt_DRBG.c

```diff
@@ -1011,10 +1011,9 @@ EverCrypt_DRBG_generate_sha1(
   {
     x1 = KRML_EABORT(Hacl_HMAC_DRBG_state, "unreachable (pattern matches are exhaustive in F*)");
   }
-  bool b;
   if (x1.reseed_counter[0U] > Hacl_HMAC_DRBG_reseed_interval)
   {
-    b = false;
+
```

Comment:
interesting side-effect of this patch, but why not -- need to improve code-gen to skip empty if blocks